com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs

```diff
@@ -647,6 +646,8 @@ public void Execute(ScriptableRenderContext context, ref RenderingData rendering
                     SortStable(m_ActiveRenderPassQueue);
                 }
 
+                if (cameraData.xr.enabled)
+                    useRenderPassEnabled = false;
```

Comment:
Does it make sense to set this here? cameraData.xr.enabled should be relatively static. Any reason why we do not do this earlier?

Reply:
So I've actually tried multiple approaches on disabling RenderPass for XR and this one was the only way it worked (was a little bit difficult because locally i didn't have a machine i could reproduce the failing tests that required this change). I think I could find a better place as this was done just to be 100% sure it's always disabled. I can address moving this to a better place in the PR i have in the works (which is waiting for the test coverage this PR introduces), is that alright with you?